hw/ip/prim/data/prim_mubi_pkg.sv.tpl

```diff
@@ -0,0 +1,156 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// This package defines common multibit signal types, active high and active low values and
+// the corresponding functions to test whether the values are set or not.
+<%
+## This Mako template can be rendered using
```

Comment:
Could you indicate here this is autogenerated? I m hoping to make a follow up PR to unify this banner for consistency across all tools that generate code.

Reply:
jep, done!